app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -131,23 +134,28 @@ fun PledgedProjectsOverviewScreen(
 
                     ppoCards[index]?.let {
                         PPOCardView(
-                            viewType = it.viewType,
+                            viewType = it.viewType() ?: PPOCardViewType.UNKNOWN,
                             onCardClick = { },
-                            onProjectPledgeSummaryClick = { onProjectPledgeSummaryClick(it.backingDetailsUrl) },
-                            projectName = it.projectName,
-                            pledgeAmount = it.pledgeAmount,
-                            imageUrl = it.imageUrl,
-                            imageContentDescription = it.imageContentDescription,
-                            creatorName = it.creatorName,
-                            sendAMessageClickAction = { onSendMessageClick(it.projectSlug) },
-                            shippingAddress = it.shippingAddress,
-                            showBadge = it.showBadge,
+                            onProjectPledgeSummaryClick = { onProjectPledgeSummaryClick(it.backingDetailsUrl() ?: "") },
+                            projectName = it.projectName(),
+                            pledgeAmount = it.amount(),
+                            imageUrl = it.imageUrl(),
+                            imageContentDescription = it.imageContentDescription(),
+                            creatorName = it.creatorName(),
+                            sendAMessageClickAction = { onSendMessageClick(it.projectSlug() ?: "") },
+                            shippingAddress = it.address()?.addressLine1() ?: "", // TODO replace with formatted address from PPO response
```

Comment:
we will need to update this with a formatted string. for now im just grabbing the shipping address from the backing object and using the first line, but this should be different when the backend piece is complete (we will either receive a formatted string or will have to format it ourselves).